Add eviction thresholds parameters (#42142)
kubernetes/website: content/en/docs/tasks/administer-cluster/kubelet-config-file.md

serializeImagePulls: false
evictionHard:
    memory.available:  "200Mi"
    nodefs.available:  "10%"
    imagefs.available: "15%"
    nodefs.inodesFree: "5%"
```

In this example, the Kubelet is configured with the following settings:

1. address: The Kubelet will serve on IP address 192.168.0.8.
2. port: The Kubelet will serve on port 20250.

> 3. serializeImagePulls: Image pulls will be done in parallel.

Reviewer: You could either:

- write something more like:
  ```markdown
  * `address` and `port`: The kubelet listens on TCP port 20250, binding to the IPv4 address 192.168.0.8 only.
  * `serializeImagePulls`: The kubelet may pull images in parallel. Because `maxParallelImagePulls` is
     not set, there is no limit on how many parallel pulls can happen at one time.
  * `evictionSoft`: Set some eviction thresholds.
  ...more details here...
  * `evictionHard`: Set more eviction thresholds.
  ```
* add similar information to the example YAML snippet, using YAML comments.
* both of the above

4. Eviction Thresholds: The Kubelet will evict Pods under the following conditions:

>    - When the node's available memory drops below 200Mi.

Reviewer: ```suggestion
   - When the node's available memory drops below 200MiB.
```

   - When the node's main filesystem's available space is less than 10%.
   - When the image filesystem's available space is less than 15%.

>    - When the node's main filesystem's free inodes are less than 5%.

Reviewer: ```suggestion
   - When more than 95% of the node's main filesystem's inodes are in use.
```
Let's assume readers can do the math here. If it makes people stop and think, I personally think that provides a benefit.

> 1.  address: The Kubelet will serve on IP address 192.168.0.8.
> 2.  port: The Kubelet will serve on port 20250.
> 3.  serializeImagePulls: Image pulls will be done in parallel.
> 4.  Eviction Thresholds: The Kubelet will evict Pods under the following conditions:
>     -   When the node's available memory drops below 200Mi.
>     -   When the node's main filesystem's available space is less than 10%.
>     -   When the image filesystem's available space is less than 15%.
>     -   When the node's main filesystem's free inodes are less than 5%.

Reviewer 2: ```suggestion
1. address: The Kubelet will serve on IP address 192.168.0.8.
2. port: The Kubelet will serve on port 20250.
3. serializeImagePulls: Image pulls will be done in parallel.
4. Eviction Thresholds: The Kubelet will evict Pods under the following conditions:
   - When the node's available memory drops below 200Mi.
   - When the node's main filesystem's available space is less than 10%.
   - When the image filesystem's available space is less than 15%.
   - When the node's main filesystem's free inodes are less than 5%.
```
